src/models/strengthMachineProfileStats.ts

```diff
@@ -0,0 +1,43 @@
+import { Model } from '../model'
+import { AuthenticatedResponse, SessionHandler } from '../session'
+
+export interface StrengthMachineProfileStatsMeta {
+  strengthMachineId: number
+}
+
+export interface StrengthMachineProfileStatsData {
+  peakPower: number
+}
+
+export interface StrengthMachineProfileStatsResponse extends AuthenticatedResponse {
+  strengthMachineProfileStats: StrengthMachineProfileStatsData
+}
+
+export class StrengthMachineProfileStats extends Model {
+  private _strengthMachineProfileStatsData: StrengthMachineProfileStatsData
+  private readonly _strengthMachineProfileStatsMeta: StrengthMachineProfileStatsMeta
+
+  constructor (strengthMachineProfileStatsData: StrengthMachineProfileStatsData, strengthMachineProfileStatsMeta: StrengthMachineProfileStatsMeta, sessionHandler: SessionHandler) {
+    super(sessionHandler)
+    this._strengthMachineProfileStatsData = strengthMachineProfileStatsData
+    this._strengthMachineProfileStatsMeta = strengthMachineProfileStatsMeta
```

Comment:
This doesn't follow any of the previous model design patterns.  Please remove `_strengthMachineProfileStatsMeta` and have the server return `strengthMachineId` in the response.